### ISSUE TITLE
Move test data to an S3 bucket in the Hubverse AWS account

### ISSUE THREAD

**Issue by bsweger** [hubverse-org/hubData#54]
The `hubData` test suite uses small cloud-based hubs to test S3 integration:

```
➜ aws s3 ls hubverse/hubutils/testhubs/ --no-sign-request
                           PRE empty/
                           PRE parquet/
                           PRE simple/
```

Now that the Hubverse has a dedicated S3 account, we can we move the above objects to it and modify the packages that use these files for testing?

**Comment by annakrystalli** [comment]
Sure! Happy to help with this. 

It would be nice if I could have access to the S3 account so I can make modifications when developing without needing assistance from others.

**Comment by bsweger** [comment]
> Sure! Happy to help with this.
> 
> It would be nice if I could have access to the S3 account so I can make modifications when developing without needing assistance from others.

Of course! I propose an S3 bucket called something like `hubverse-test-data` that hubverse devs have read/write access to. Will ping the team to weigh in.

**Comment by nickreich** [comment]
No concerns! This seems like a sensible idea.
